A/Antic/build_tarballs.jl

```diff
@@ -38,7 +42,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="FLINT_jll", uuid="e134572f-a0d5-539d-bddf-3cad8db41a82"))
+    Dependency(PackageSpec(name="FLINT_jll", uuid="e134572f-a0d5-539d-bddf-3cad8db41a82", version=v"0.0.1+0"))
```

Comment:
Don't use build numbers, `Pkg` doesn't like them:
```suggestion
    Dependency(PackageSpec(name="FLINT_jll", uuid="e134572f-a0d5-539d-bddf-3cad8db41a82", version=v"0.0.1"))
```

Reply:
How does it distinguish between `+0` and `+1`?

Reply:
Oops, didn't see this message before merging.  It doesn't, it'll grab the latest one :grimacing:

Reply:
I won't matter here. It was just curiosity.